lib/rules/cli/4.1.yaml

```diff
@@ -1372,3 +1372,26 @@
   :options:
     :skip-check: --skip-check=<value>
     :z: -z default
+:create_cronjob:
+  :cmd: oc create cronjob <name>
+  :options:
+    :allow_missing_template_keys: --allow-missing-template-keys=<value>
+    :dry_run: --dry-run=<value>
+    :image: --image=<value>
+    :output: --output <value>
+    :template: --template=<value>
+    :restart: --restart=<value>
+    :save_config: --save-config=<value>
+    :schedule: --schedule=<value>
+    :validate: --validate=<value>
+:create_job:
+  :cmd: oc create job <name>
+  :options:
+    :allow_missing_template_keys: --allow-missing-template-keys=<value>
+    :dry_run: --dry-run=<value>
+    :from: --from=<value>
+    :image: --image=<value>
+    :output: --output <value>
+    :template: --template=<value>
+    :save_config: --save-config=<value>
+    :validate: --validate=<value>
```

Comment:
Please sort the command and options in alphanumeric order

Reply:
Done